src/topic_training/src/Begin_Here.md

```diff
@@ -0,0 +1,122 @@
+# WRoverPlayground - Topic Training
+Now that you have completed the Tools Training, you are ready to learn about ROS topics. From the Setup Training, you should be familiar with:
+- the basics of ROS Nodes
+- rosnode
+- and roscore.
```

Comment:
Here if you are referring to the commands `rosnode` and `roscore`, use \`\` around them so there's less ambiguity.
```suggestion
- `rosnode`
- and `roscore`.
```

Reply:
Fixed in 7ece46c.